src/content-helper/dashboard-widget/components/top-posts.tsx

```diff
@@ -3,31 +3,33 @@
  */
 import { Spinner } from '@wordpress/components';
 import { useEffect, useState } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import { ContentHelperError } from '../../../content-helper/common/content-helper-error';
+import { ContentHelperError } from '../../common/content-helper-error';
+import { TopPostData } from '../model';
 import { DashboardWidgetProvider } from '../provider';
-import { TopPostListItem } from './component-list-item';
-import { TopPostData } from './model';
+import { TopPostsFilters } from './top-posts-filters';
+import { TopPostsList } from './top-posts-list';
 
 const FETCH_RETRIES = 1;
 
 /**
  * List of the top posts.
  */
-export function TopPostList() {
+export function TopPosts() {
 	const [ loading, setLoading ] = useState<boolean>( true );
 	const [ error, setError ] = useState<ContentHelperError>();
 	const [ posts, setPosts ] = useState<TopPostData[]>( [] );
+	const [ period, setPeriodFilter ] = useState<string>( '7' );
+	const [ metric, setMetricFilter ] = useState<string>( 'views' );
```

Comment:
If we use enums then we can implement strong typing here and above.

Reply:
Will do.